specs/core/0_beacon-chain.md

```diff
@@ -710,8 +710,8 @@ def shuffle(values: List[Any], seed: Bytes32) -> List[Any]:
             if remaining == 1:
                 break
 
-            # Read 3-bytes of `source` as a 24-bit big-endian integer.
-            sample_from_source = int.from_bytes(source[position:position + rand_bytes], 'big')
+            # Read 3-bytes of `source` as a 24-bit little-endian integer.
+            sample_from_source = int.from_bytes(source[position:position + rand_bytes], 'little')
```

Comment:
```suggestion
            sample_from_source = int_to_bytes3(source[position:position + rand_bytes])
```

Reply:
We want to convert bytes-to-int, not int-to-bytes. `int.from_bytes` is correct, but definitely very pythonic

Reply:
Oooh, sorry for the misreading.
And I added `bytes_to_int` in #576.